### DISCUSSION TITLE
accent color?

### DISCUSSION THREAD

**Discussion by DenDanskeMine** [retypeapp/retype#635] [Q&A] [answered]
## Hello :)

Is there a option to customize the accent colors?
The `blue` default isn’t a perfect match for my branding :(
I could see from some other posts back in 2021 that you where working on it.
but i cant find the docu for it

btw loving the product <3

**Comment by geoffreymcgill** [comment, marked as answer]
As of yet, we have not been able to build the functionality to change the theme color. It's still definitely something we would like to add, but there is no specific time frame at this point.

Closing as this is a duplicate and we should keep all the discussion regarding this topic centralized in one thread, see https://github.com/retypeapp/retype/discussions/282.

> btw loving the product <3

Awesome, thanks for the feedback!
